svm/src/rollback_accounts.rs

```diff
@@ -99,6 +99,15 @@ impl RollbackAccounts {
         }
     }
 
+    /// Return a reference to the fee payer account.
+    pub fn fee_payer(&self) -> &TransactionAccount {
```

Comment:
nit: we only use this to get the balance currently, should we change this to `fee_payer_balance` or add another method for the balance to avoid the slightly awkward need for the caller to do `fee_payer().1.lamports()`?